src/Select.js

```diff
@@ -143,11 +143,7 @@ class Select extends React.Component {
 	}
 
 	componentWillUnmount () {
-		if (!document.removeEventListener && document.detachEvent) {
-			document.detachEvent('ontouchstart', this.handleTouchOutside);
```

Comment:
`detachEvent` is for supporting IE8 here. However, IE8 doesn't support touch events, right?

Reply:
Yeah, you're right about this.